index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-AC">AC</button>
+      <button class="calculator-button calculator-positive-negative">+/-</button>
+      <button class="calculator-button calculator-percentage">%</button>
+      <button class="calculator-button calculator-divide">÷</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-seven">7</button>
+      <button class="calculator-button calculator-eight">8</button>
+      <button class="calculator-button calculator-nine">9</button>
+      <button class="calculator-button calculator-multiply">×</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-four">4</button>
+      <button class="calculator-button calculator-five">5</button>
+      <button class="calculator-button calculator-six">6</button>
+      <button class="calculator-button calculator-subtract">-</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-one">1</button>
+      <button class="calculator-button calculator-two">2</button>
+      <button class="calculator-button calculator-three">3</button>
+      <button class="calculator-button calculator-add">+</button>
+    </div>
+    <div class="calculator-row calculator-row-last">
```

Comment:
If you decide to go with the __modifier naming convention, calculator-row__last would be another place to use it.